Sources/AuthFoundation/Responses/OAuth2ServerError.swift

```diff
@@ -33,3 +33,35 @@ public struct OAuth2ServerError: Decodable, Error, LocalizedError {
         case description = "errorDescription"
     }
 }
+
+extension OAuth2ServerError {
+    ///  Possible  OAuth 2.0 server error code
+    public enum OAuth2ServerErrorCode: String, Decodable {
```

Comment:
This name feels a bit verbose. I'd suggest finding something shorter, like "Code".  The rest is implied, since it's already within the `OAuth2ServerError` namespace, so repeating it in the enum name seems redundant.

Reply:
Good callout on this. I agree with you, I'll make the changes.